about/contribute.html

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "Contributing"
+permalink: /about/contribute/
```

Comment:
Is there are reason for the disagreement between title and file name?

Reply:
Just me being indecisive 😆 
I'll switch everything to `contributing`